legend-engine-core/legend-engine-core-base/legend-engine-core-language-pure/legend-engine-language-pure-compiler/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/PackageableElementFirstPassBuilder.java

```diff
@@ -85,6 +86,10 @@ public PackageableElementFirstPassBuilder(CompileContext context)
     @Override
     public PackageableElement visit(org.finos.legend.engine.protocol.pure.v1.model.packageableElement.PackageableElement element)
     {
+        if (element instanceof Database)
```

Comment:
Shouldn't this happen in the compiler for the relational grammar? The grammar for the relational parser has been changed in this PR - I'd expect the compiler change to also happen in that module.